### PR TITLE
fixed security hotspot

### PR DESCRIPTION
**Pull request by kapoliub**

### CODE REVIEW

File: modules/payment/payment.service.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const axios = require('axios');
 const CloudIpsp = require('cloudipsp-node-js-sdk');
 const crypto = require('crypto');
 const {
-  CRYPTO,
+  // CRYPTO,
```
Comment (moran711) [inline modules/payment/payment.service.js:5]: Please delete this line

File: server.js [hunk 2]
```diff
@@ -69,12 +69,13 @@ const server = new ApolloServer({
 
 const PORT = process.env.PORT || 5000;
 
-const app = express();
+const app1 = express();
```
Comment (moran711) [inline server.js:72]: please rename it to more suitable name

File: modules/payment/payment.service.js (hunk 1 above)
Comment (kapoliub) [reply modules/payment/payment.service.js]: fixed

File: server.js (hunk 2 above)
Comment (kapoliub) [reply server.js]: fixed

**Pull request merged**